verticaltabs.jsm

```diff
@@ -417,7 +455,7 @@ VerticalTabs.prototype = {
       toolbar._toolbox = null; // reset value set by constructor
       toolbar.removeAttribute('toolboxid');
       toolbar.removeAttribute('collapsed');
-      toolbar.removeChild(spacer);
+      toolbar.removeChild(find_input);
```

Comment:
Do we also need to remove the `hidden_tab`?  (Perhaps only sometimes?  If only sometimes, would it be better to always have it in the tabs, and hide/show it with visibility?)